patterns-modules/design-patterns-creational-2/pom.xml

```diff
@@ -16,9 +16,13 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-inline</artifactId>
-            <version>${mockito.version}</version>
+            <version>${mockito-inline.version}</version>
             <scope>test</scope>
         </dependency>
     </dependencies>
+    
+    <properties>
+        <mockito-inline.version>5.2.0</mockito-inline.version>
```

Comment:
Same here if possible please put in the main pom.xml

Reply:
changes done